--- a/plextraktsync/pytrakt_extensions.py
+++ b/plextraktsync/pytrakt_extensions.py
@@ -21,7 +21,7 @@
         completed=False,
         last_watched_at=None,
         collected_at=None,
-        metadata=None,
+        **kwargs,
     ):
         self.number = number
         self.aired = aired
@@ -36,7 +36,7 @@
 
 
 class SeasonProgress:
-    def __init__(self, number=0, title=None, aired=0, completed=False, episodes=None, metadata=None):
+    def __init__(self, number=0, title=None, aired=0, completed=False, episodes=None, **kwargs):
         self.number = number
         self.aired = aired
         self.episodes = {}
@@ -74,7 +74,7 @@
         next_episode=0,
         last_episode=0,
         last_collected_at=None,
-        metadata=None,
+        **kwargs,
     ):
         self.aired = aired
         self.last_watched_at = last_watched_at
