diff --git a/plextraktsync/pytrakt_extensions.py b/plextraktsync/pytrakt_extensions.py
index 4f65789b..fdca4f82 100644
--- a/plextraktsync/pytrakt_extensions.py
+++ b/plextraktsync/pytrakt_extensions.py
@@ -27,6 +27,7 @@ def __init__(
         completed=False,
         last_watched_at=None,
         collected_at=None,
+        **kwargs,
     ):
         self.number = number
         self.aired = aired
@@ -41,7 +42,7 @@ def get_completed(self):
 
 
 class SeasonProgress:
-    def __init__(self, number=0, title=None, aired=0, completed=False, episodes=None):
+    def __init__(self, number=0, title=None, aired=0, completed=False, episodes=None, **kwargs):
         self.number = number
         self.aired = aired
         self.episodes = {}
@@ -78,6 +79,7 @@ def __init__(
         next_episode=0,
         last_episode=0,
         last_collected_at=None,
+        **kwargs,
     ):
         self.aired = aired
         self.last_watched_at = last_watched_at
